niilovuo/ECSE428_Master_Chef
# Add automated test coverage

Close #5 

Adds the following functionality:
* Add dependency `pytest-cov` for generating coverage
* Run `pytest` on every build
* Report coverage in the build logs (see the step `Run tests` under the `build` job)
* Report coverage as a comment of a PR (you'll see below)

Notes:
* Add a dummy test just so it has something to test (needs to be deleted eventually)
* Add a very basic project structure (it's pretty easy to change this though)
* ***Not*** make builds fail if coverage is not high enough (also pretty easy to change)
* ***Not*** add postgresql stuff (should be separate PR)
